ksqldb-common/src/test/java/io/confluent/ksql/util/WelcomeMsgUtilsTest.java

```diff
@@ -20,20 +20,19 @@
 
 import java.io.PrintWriter;
 import java.io.StringWriter;
-import org.easymock.EasyMock;
-import org.easymock.EasyMockRunner;
-import org.easymock.Mock;
-import org.easymock.MockType;
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
 
-@RunWith(EasyMockRunner.class)
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
for some reason EasyMock wasn't working as well after the JDK upgrade. instead of trying to figure it out I just replaced the failing EasyMock tests with Mockito

Reply:
I assume that we're all in on Mockito at this point anyway. I haven't seen anyone write an Easymock test and assume there aren't many.